### ISSUE TITLE
Make it possible to setup content and media starting nodes in the user invite flow

### ISSUE THREAD

**Issue by BatJan** [umbraco/Umbraco-CMS#3114]
When using the invite function it's possible to assign a user to a user group. But it's not possible to pick starting nodes for content and media. This can only be done once the user is invited. 

Therefore I suggest we should make it possible to add the start nodes as a part of the invite flow so it feels less "clunky" and more straight forward.

Now I don't know if there are any technical reasons that makes this impossible currently. Just an idea since it was a bit time consuming sending invites out for 24+ users this past weekend 😄

**Comment by BatJan** [comment]
Hey @kjac thanks for chiming in :)

The upsite on how it currently works is that it's fast to setup users who can easily be assigned a user group, which is great. And playing a bit around with this I have also been having second thoughts about whether making it possible to override the default start nodes makes sense in other use cases.

I like how quick and simple the current UI is and adding the two suggested fields will of course add a bit more bloat to the UI initially. So if it's a matter of setting up 1 or 2 users at a time then it's probably not worth adding these two extra options up front.

But when I think about the hassle it can be for a webmaster on a local government site to setup maybe more than 10 users at a time then it can be quite time consuming having to first send out all the invites and then afterwards find the users and setup their starting nodes.

So perhaps instead of showing it up front a more discrete option could be added that when triggered will reveal them? Does that make sense? Some kind of discrete text or button so it does not appear as a mandatory thing but an optional one.

**Comment by nul800sebastiaan** [comment]
Hey @BatJan - I totally get why this would be a requirement in your specific case and I feel your pain 😅

However, this is also a very unique requirement you have for this site and I don't see this as a problem most users of Umbraco will encounter. 🙂

Ps. Yay, 24 days in Umbraco is coming! 😍

**Comment by BatJan** [comment]
@nul800sebastiaan - Well, as I mentioned in my previous comment I can think of some webmasters in local governments and probably bigger corporations who would have this issue too. Not just me. But doing 24 days invites meant I got to do some dogfooding - So now I know what it can potentially feel like to manage this at a larger scale. 

So even though it might not have been communicated clearly in my initial post this was not a "me" issue, which I would like to change just for the sake of me inviting people to 24 days once a year. 😆  

So to clarify - I raised because I feel the pain of webmaster and content managers who are sometimes in the same situation 😃 

Yes the entire 24 days team and the authors are looking very much forward to December! 🎅
